client/tailscale/localclient.go

```diff
@@ -903,3 +903,43 @@ func (r jsonReader) Read(p []byte) (n int, err error) {
 	}
 	return r.b.Read(p)
 }
+
+// ProfileStatus returns the current profile and the list of all profiles.
+func (lc *LocalClient) ProfileStatus(ctx context.Context) (current ipn.LoginProfile, all []ipn.LoginProfile, err error) {
+	body, err := lc.send(ctx, "GET", "/localapi/v0/profiles/current", 200, nil)
+	if err != nil {
+		return
+	}
+	current, err = decodeJSON[ipn.LoginProfile](body)
+	if err != nil {
+		return
+	}
+	body, err = lc.send(ctx, "GET", "/localapi/v0/profiles/", 200, nil)
+	if err != nil {
+		return
+	}
+	all, err = decodeJSON[[]ipn.LoginProfile](body)
+	return current, all, err
+}
+
+// NewProfile creates and switches to a new unnamed profile. The new profile is
+// not assigned an ID until it is persisted after a successful login.
+// In order to login to the new profile, the user must call LoginInteractive.
+func (lc *LocalClient) NewProfile(ctx context.Context) error {
```

Comment:
NewT makes it sounds like it returns T.

How about `SwitchToEmptyProfile` or something?

Reply:
Done.